packages/math-input/src/components/input/math-input.tsx

```diff
@@ -908,11 +908,21 @@ class MathInput extends React.Component<Props, State> {
                 overrides.css. */}
                 <div
                     className="keypad-input"
-                    // @ts-expect-error - TS2322 - Type 'string' is not assignable to type 'number | undefined'.
-                    tabIndex={"0"}
+                    tabIndex={0}
                     ref={(node) => {
                         this.inputRef = node;
                     }}
+                    onFocus={() => {
+                        this.focus();
+                    }}
+                    onBlur={() => {
```

Comment:
I will be making a separate ticket and PR that adds logic to ensure that the mathInput retains the focus styles when any element within the keypad is the activeElement. Unfortunately Storybook really breaks being able to test "activeElement" due to the use of some strange iframes, so we've decided to tackle this issue post launch.

Reply:
https://khanacademy.atlassian.net/browse/LC-1209